src/Pages/components/Searchbar.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react'
+import { v4 as uuid } from 'uuid'
+import styled from 'styled-components/macro'
+import useEvents from '../EventMain/useEvents'
+import Input from './common/Input'
+
+export default function Searchbar({
+  handleEventSuggestion,
+  handleEventSearch,
+}) {
+  const { eventArray } = useEvents()
+  const eventSuggestions = eventArray.map((event) => event.name)
+  const [filteredSuggestions, setFilteredSuggestions] = useState([])
+  const [eventNameInput, setEventNameInput] = useState('')
+
+  return (
+    <>
+      <EventSearchbar
+        placeholder="search..."
+        value={eventNameInput}
+        onKeyUp={handleInputKeyUp}
+        onChange={(e) => setEventNameInput(e.target.value)}
+      />
+      <SuggestionContainer>
+        <SuggestionList>
+          {filteredSuggestions.map((sug) => (
+            <SuggestionItems
+              key={uuid()}
+              onClick={() => handleEventSuggestionClick(sug)}
+            >
+              {getMarkedSuggestionText(sug)}
+            </SuggestionItems>
+          ))}
+        </SuggestionList>
+      </SuggestionContainer>
+    </>
+  )
+  function handleInputKeyUp(event) {
+    const inputVal = event.target.value
+    if (event.key === 'Enter') {
```

Comment:
I used to have some problems with keyboard keys on mobile devices. Make sure to test this on Android and Apple devices to see if it works with their software keyboards.

Reply:
I just habe android devices, will see this  after the QA test